### ISSUE TITLE
Does not Compile (Texlive, Windows)

### ISSUE THREAD

**Issue by bempelise** [SublimeText/LaTeXTools#358]
I have setup LaTeXTools as follows:

```
    "windows": {
        // Path used when invoking tex & friends; "" is fine for MiKTeX
        // For TeXlive 2011 (or other years) use
        "texpath" : "C:\\dev\\texlive\\2013\\bin\\win32;$PATH",
        // "texpath" : "",
        // TeX distro: "miktex" or "texlive"
        "distro" : "texlive"
    },
```

and when I compile I just get:

```
TraditionalBuilder: Invoking latexmk... 
```

and in the Sublime console:

```
['latexmk', '-cd', '-e', "$pdflatex = 'pdflatex -interaction=nonstopmode -synctex=1 %S %O'", '-f', '-pdf', u'hello.tex']
Exception in thread Thread-3:
Traceback (most recent call last):
  File ".\threading.py", line 532, in __bootstrap_inner
  File ".\makePDF.py", line 110, in run
LookupError: unknown encoding: 737
```

Any ideas?

**Comment by ModarTensai** [comment]
Thanks @bempelise but that is unfortunate. I have the same problem but with another encoding (Arabic "720") and I don't want to change that setting. 

Here is the same message:
`LookupError: unknown encoding: 720`

I checked `%Python_Path%/Lib/encodings/` folder and it has the encoding file but the name is `cp720` instead of `720`.  So I tried to add an alias to `%Python_Path%/Lib/encodings/aliases.py` but that didn't work for some reason (maybe I needed a restart).

However, I temporarily changed line 318 in `Sublime TExt 3/Packages/LaTeXTools/makePDF.py` from:
`self.caller.builder.set_output(out.decode(self.caller.encoding,"ignore"))`
to 
`self.caller.builder.set_output(out.decode('cp'+self.caller.encoding,"ignore"))`
and it worked magically. 

I hope this get fixed in the next update patch.
